mlflow/utils/virtualenv.py

```diff
@@ -281,9 +281,7 @@ def _create_virtualenv(
 
                 tmp_req_file = f"requirements.{uuid.uuid4().hex}.txt"
                 Path(tmpdir).joinpath(tmp_req_file).write_text("\n".join(deps))
-                cmd = _join_commands(
-                    activate_cmd, f"python -m pip install --quiet -r {tmp_req_file}"
-                )
+                cmd = _join_commands(activate_cmd, f"python -m pip install -r {tmp_req_file}")
```

Comment:
Can we also make `pyenv install` more verbose ?

Reply:
Why?

Reply:
rethinking it, `pyenv install --verbose` might be too verbose, current code should be good.